clear_generated_images.py

```
@@ -0,0 +1,186 @@
+"""
+Clear Generated Images Folder
+
+This script safely clears the generated_images folder and all its contents.
+Useful for starting fresh with a new pipeline run.
+Handles Windows permission issues and stubborn files.
+"""
+
+import os
+import shutil
+import stat
+import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)